src/sql/workbench/contrib/notebook/test/browser/notebookLinkHandler.test.ts

```diff
@@ -103,4 +103,31 @@ suite('Noteboook Link Handler', function (): void {
 		result = new NotebookLinkHandler(notebookUri, Object.assign(document.createElement('a'), { href: '/tmp/my%20stuff.png' }), configurationService);
 		assert.strictEqual(result.getLinkUrl(), `.${path.sep}my%2520stuff.png`, 'Basic link test with %20 filename failed');
 	});
+
+	test('Should return correctly encoded url/filePath', () => {
+		test('when given an already-encoded URL', () => {
+			let notebookLinkHandler = new NotebookLinkHandler(notebookUri, 'https://github.com/search/advanced?q=test&r=microsoft%2Fazuredatastudio&type=Code', configurationService);
+			assert.strictEqual(notebookLinkHandler.getEncodedLinkUrl(), `https://github.com/search/advanced?q=test&r=microsoft%2Fazuredatastudio&type=Code`, 'HTTPS link does not need encoding');
+		});
+
+		test('when given an already encoded URL with non-reserved characters', () => {
+			let notebookLinkHandler = new NotebookLinkHandler(notebookUri, 'https://github.com/search/advanced?q=test&r=(microsoft%2Fazuredatastudio)&type=Code', configurationService);
+			assert.strictEqual(notebookLinkHandler.getEncodedLinkUrl(), `https://github.com/search/advanced?q=test&r=(microsoft%2Fazuredatastudio)&type=Code`, '() in HTTP link failed to resolve');
```

Comment:
Not sure what "resolve" is supposed to mean here.

and in general for all of these it's probably fine just to just remove the message completely. It doesn't really add much information since the test is extremely simple and each one is now only testing 1 thing.